internal/eliza/globals.go

```diff
@@ -289,3 +289,22 @@ var reflectedWords = map[string]string{
 	"you":    "me",
 	"me":     "you",
 }
+
+var introResponses = []string{
+	"Hi %s.  I'm Eliza.",
```

Comment:
Nit: one space between sentences, please.

https://www.theatlantic.com/technology/archive/2011/01/why-you-should-never-ever-use-two-spaces-between-sentences/69579/

Reply:
Changed.  Although I counter this with:

https://www.theatlantic.com/entertainment/archive/2011/01/you-can-have-my-double-space-when-you-pry-it-from-my-cold-dead-hands/69592/